pkg/nmstate/policylist.go

```diff
@@ -29,7 +42,7 @@ func ListPolicy(apiClient *clients.Settings, options ...goclient.ListOptions) ([
 	glog.V(100).Infof(logMessage)
 
 	policyList := &nmstateV1.NodeNetworkConfigurationPolicyList{}
-	err := apiClient.Client.List(context.TODO(), policyList, &passedOptions)
+	err = apiClient.Client.List(context.TODO(), policyList, &passedOptions)
 
 	if err != nil {
 		glog.V(100).Infof("Failed to list NodeNetworkConfigurationPolicy due to %s", err.Error())
```

Comment:
Does the apiClient on line 58 need to be updated to `apiClient.Client`?

Reply:
+1 fixed.